src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheck.java

```diff
@@ -122,6 +122,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
```

Comment:
this one a bit more complicated ...

please add `To configure the default check` and add default example as we did gsoc-first-issue.

Reply:
There are various example configs already present in the lines below. This line is just used for signifying the start of examples section.

Reply:
ok, let it be, we will address it later